plugins/quetz_tos/quetz_tos/api.py

```diff
@@ -33,11 +33,15 @@ def post_file(file):
     return file.filename
 
 
-@router.get("/api/tos", tags=['Terms of Service'])
-def get_current_tos(db: Session = Depends(get_db)):
+@router.get("/api/tos/?{lang}", tags=['Terms of Service'])
```

Comment:
I guess this does not work?

For adding query parameters you can look at how it is done here: https://github.com/tiangolo/fastapi#create-it. See how they have the `"q"` query parameter.

So you probably want something like:
```python
@router.get("/api/tos", tags=['Terms of Service'])
def get_current_tos(lang: str = "EN", db: Session = Depends(get_db)):
```

And then this API will be called with `GET /api/tos?lang=FR`